### PR TITLE
Adds mande chain

### PR DESCRIPTION
**Pull request by prtk418**

### CODE REVIEW

File: mande/chain.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mande",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://mande.network",
+  "pretty_name": "Mande Network",
+  "chain_id": "mande_18071918-1",
+  "bech32_prefix": "mande",
```
Comment (JeremyParish69) [inline mande/chain.json:9]: Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: mande/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      "socials": {
+        "website": "https://mande.network/",
+        "twitter": "https://twitter.com/MandeNetwork"
+      },
+      "keywords": [
+        "credibility",
+        "web3"
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.png",
```
Comment (JeremyParish69) [inline mande/assetlist.json:38]: Can I request the png so be made a bit smaller to reduce it's file size? ~3000x3000 is quite huge....
Also, can we please put this in a square canvas so the image is 1:1 AR?

File: mande/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      "type_asset": "sdk.coin",
+      "base": "amand",
+      "name": "Mande Network",
+      "display": "mand",
+      "symbol": "MAND",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.svg"
+        }
```
Comment (JeremyParish69) [inline mande/assetlist.json:27]: Please add theme::primary_color_hex

File: mande/chain.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.svg"
+  },
+  "description": "Credibility Hub for Web3",
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mande/images/mande.svg"
+    }
```
Comment (JeremyParish69) [inline mande/chain.json:40]: Please add theme::primary_color_hex

File: mande/chain.json (hunk 1 above)
Comment (prtk418) [reply mande/chain.json:9]: Raised a PR - https://github.com/satoshilabs/slips/pull/1753

File: mande/assetlist.json (hunk 2 above)
Comment (prtk418) [reply mande/assetlist.json:42]: Done, added 500X500

File: mande/chain.json (hunk 1 above)
Comment (prtk418) [reply mande/chain.json:9]: This is merged @JeremyParish69

**Pull request merged**